spec/functions/extlib/cache_data_spec.rb

```diff
@@ -14,14 +16,14 @@
   describe 'data caching' do
     context 'when not in cache' do
       it 'returns supplied data' do
-        is_expected.to run.with_params(namespace, data_name, initial_data).and_return(initial_data)
+        expect(subject).to run.with_params(namespace, data_name, initial_data).and_return(initial_data)
```

Comment:
This is new and surprises me. I think this is https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecimplicitexpect but not 100% sure. Why show a difference in line 10/11 (which also appears to be the exact same test)?